Expensify/App
# Upgrade expensify-common

### Details

Made some changes to ExpensiMark that were recently merged and deployed to npm, so this PR upgrades expensify-common so we can have those new changes.

If you're confused as to why the package.json isn't using the commit hash anymore, it's because I followed these steps (which I think use the major/minor version): https://stackoverflow.com/c/expensify/questions/7216

### Fixed Issues
N/A, but here's the expensify-common PR: https://github.com/Expensify/expensify-common/pull/376

### Tests
Same as QA

### QA Steps
Try to send this as a message in any chat, and verify it renders the same as the screenshot:

> \```hi\```
\```hi
\```
\```
hi\```
\```
hi
\```

<img width="821" alt="Screen Shot 2021-05-24 at 8 38 41 PM" src="https://user-images.githubusercontent.com/31285285/119348894-09e2a580-bcd0-11eb-8ef0-3227acfdfb5c.png">

### Tested On


### Screenshots

### Web
https://user-images.githubusercontent.com/31285285/119344670-84102b80-bcca-11eb-9903-8ac3907aad98.mp4

### Mobile Web
https://user-images.githubusercontent.com/31285285/119344719-90948400-bcca-11eb-80b9-5bc40a1ac858.mp4

### iOS
https://user-images.githubusercontent.com/31285285/119344749-99855580-bcca-11eb-84ac-090692a7aaf6.mp4

### Android
https://user-images.githubusercontent.com/31285285/119435608-0ccfab80-bd4d-11eb-9e4e-62fbabdb5358.mp4

### Desktop
https://user-images.githubusercontent.com/31285285/119344810-af931600-bcca-11eb-908f-69747c4200ff.mp4
